lib/src/check_internet.dart

```diff
@@ -175,4 +175,22 @@ class CheckInternet {
     _statusController.close();
     _timerHandle?.cancel();
   }
+
+  ///Ping a custom domain address and return true if it get successful response,
+  /// return false if it gets failure response
+  /// Sample Input: Uri.parse("https://google.com")
+  /// Sample Output: true
+  Future<bool> pingCustomDomainAddress({required Uri uri}) async {
```

Comment:
@msarkrish really really sorry, for the delay could we add another parameter in the function so, that user can specify exact statusCode they want to match instead of just 200

Reply:
No Problem @Vedsaga , I will add another parameter to pass statuscode.

Reply:
@Vedsaga I have added optional parameter to pingCustomDomainAddress method to pass statuscode.